mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -406,6 +406,36 @@ def run_delete_project(project_id, project_ids):
         click.echo("Invalid input")
 
 
+@cli.command("set-tileserver-api-key")
+@click.option(
+    "--project-id",
+    "-i",
+    help="Set api key for project with given project id",
+    type=str,
+)
+@click.option(
+    "--project-ids",
+    cls=PythonLiteralOption,
+    default="[]",
+    help=(
+        "Set api key for multiple projects. "
+        "Provide project id strings as a list: "
+        """'["project_a", "project_b"]'"""
+    ),
+)
+@click.option("--api-key", help="the new api key to use", type=str, required=True)
+def run_set_tileserver_api_key(project_id, project_ids, api_key):
```

Comment:
```suggestion
def run_set_tileserver_api_key(project_id: str, project_ids: Optional[list[str]], api_key: str) -> None:
```
You'll need to `from typing import Optional` as well

Reply:
I think this is a bit more complex. Here both parameters are optional `project_id`  and `project_ids`. But one of the two needs to be defied. we check for this at the beginning of the function.